### DISCUSSION TITLE
A built-in support for Chinese characters.

### DISCUSSION THREAD

**Discussion by lucasjinreal** [emilk/egui#2691] [Q&A]
Hello, attach a tiff file to program seems not a good way, Is there a way to call system fonts to support Chinese font?

**Comment by lictex** [comment]
it depends on the os. or try [font-kit](https://crates.io/crates/font-kit) for a crossplatform system font lookup. however this could be a bit tricky for some edge cases (missing fonts, weird system configurations etc.)... actually imo bundling font files into app isnt always a bad choice
